src/workflows/l10n.js

```diff
@@ -7,26 +7,26 @@ module.exports = {
 		run.fetchUpstream,
 		run.createOrCheckoutBranch,
 		run.gitMergeUpstreamBranch,
-		run.gitMergeUpstreamMasterNoFF,
+		run.gitMergeUpstreamDefaultBranchNoFF,
```

Comment:
We could probably totally remove this whole flow... maybe not this time, but in another PR